src/coreclr/jit/emit.cpp

```diff
@@ -822,6 +831,60 @@ insGroup* emitter::emitSavIG(bool emitAdd)
     }
 #endif
 
+#ifdef FEATURE_LOOP_ALIGN
+    // Did we have any align instructions in this group?
+    if (emitCurIGAlignList)
```

Comment:
In case of non-adaptive, we can have multiple `align` instructions that all represent a single padding. E.g. for non-adaptive 32B, there will be 3 align instructions
- align (15)
- align (15)
- align (1)